src/macaron/dependency_analyzer/cyclonedx.py

```diff
@@ -728,6 +744,8 @@ def get_deps_from_sbom(self, sbom_path: str | Path, target_component: Component)
             The path to the SBOM file.
         target_component: Component
             The analyzed target software component.
+        recursive: bool
+            Set to False to get the direct dependencies only (default).
```

Comment:
Missed one.

Reply:
Oops. I have fixed it here - https://github.com/oracle/macaron/pull/840/commits/eba8d7d674414df9898ac7ce8c1c7d7fbb1a98ee